controllers/akodeploymentconfig/akodeploymentconfig_controller_avi_phase.go

```diff
@@ -308,39 +308,44 @@ func (r *AKODeploymentConfigReconciler) reconcileAviInfraSetting(
 	return res, r.Update(ctx, aviInfraSetting)
 }
 
-func (r *AKODeploymentConfigReconciler) createAviInfraSetting(adc *akoov1alpha1.AKODeploymentConfig) *akov1alpha1.AviInfraSetting {
+func (r *AKODeploymentConfigReconciler) createAviInfraSetting(adc *akoov1alpha1.AKODeploymentConfig) *akov1beta1.AviInfraSetting {
 	// ShardVSSize describes ingress shared virtual service size, default value is SMALL
 	shardSize := "SMALL"
+	//TODO(chenlin): Get t1LR from akoDeploymentConfig
+	t1LR := "temp-t1LR"
 	if adc.Spec.ExtraConfigs.IngressConfigs.ShardVSSize != "" {
 		shardSize = adc.Spec.ExtraConfigs.IngressConfigs.ShardVSSize
 	}
 
-	vipNetwork := []akov1alpha1.AviInfraSettingVipNetwork{{
+	vipNetwork := []akov1beta1.AviInfraSettingVipNetwork{{
 		NetworkName: adc.Spec.ControlPlaneNetwork.Name,
 		Cidr:        adc.Spec.ControlPlaneNetwork.CIDR,
 	}}
 	//Use V6Cidr field if cidr of control plane network is IPv6
 	if utils.GetIPFamilyFromCidr(adc.Spec.ControlPlaneNetwork.CIDR) == "V6" {
-		vipNetwork = []akov1alpha1.AviInfraSettingVipNetwork{{
+		vipNetwork = []akov1beta1.AviInfraSettingVipNetwork{{
 			NetworkName: adc.Spec.ControlPlaneNetwork.Name,
 			V6Cidr:      adc.Spec.ControlPlaneNetwork.CIDR,
 		}}
 	}
 
-	return &akov1alpha1.AviInfraSetting{
+	return &akov1beta1.AviInfraSetting{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: haprovider.GetAviInfraSettingName(adc),
 		},
-		Spec: akov1alpha1.AviInfraSettingSpec{
-			SeGroup: akov1alpha1.AviInfraSettingSeGroup{
+		Spec: akov1beta1.AviInfraSettingSpec{
+			SeGroup: akov1beta1.AviInfraSettingSeGroup{
 				Name: adc.Spec.ServiceEngineGroup,
 			},
-			Network: akov1alpha1.AviInfraSettingNetwork{
+			Network: akov1beta1.AviInfraSettingNetwork{
 				VipNetworks: vipNetwork,
 			},
-			L7Settings: akov1alpha1.AviInfraL7Settings{
+			L7Settings: akov1beta1.AviInfraL7Settings{
 				ShardSize: shardSize,
 			},
+			NSXSettings: akov1beta1.AviInfraNSXSettings{
```

Comment:
Can we just remove this `NSXSettings` struct?

Reply:
Try to create an adc in testbed, stills got error
```
2023-10-23T09:49:03.003Z	ERROR	controllers.AKODeploymentConfig	failed to reconcile AKODeploymentConfig	{"AKODeploymentConfig": {"name":"install-ako-for-test-aviinfrasetting"}, "error": "AviInfraSetting.ako.vmware.com \"install-ako-for-test-aviinfrasetting-ais\" is invalid: spec.nsxSettings.t1lr: Required value", "errorCauses": [{"error": "AviInfraSetting.ako.vmware.com \"install-ako-for-test-aviinfrasetting-ais\" is invalid: spec.nsxSettings.t1lr: Required value", "errorCauses": [{"error": "AviInfraSetting.ako.vmware.com \"install-ako-for-test-aviinfrasetting-ais\" is invalid: spec.nsxSettings.t1lr: Required value"}]}]}
```